src/main/java/io/cdap/plugin/salesforce/plugin/SalesforceConnectorConfig.java

```diff
@@ -181,7 +181,10 @@ private void validateConnection() {
     try {
       SalesforceConnectionUtil.getPartnerConnection(this.getAuthenticatorCredentials());
     } catch (ConnectionException e) {
-      throw new RuntimeException("There was issue communicating with Salesforce. " + e.getMessage(), e);
+      throw new RuntimeException(
```

Comment:
For ConnectionException, it is not returning any value in e.getMessage.
Actual error message can be called using its child class's method e.getExceptionMessage(), most of the classes which extend ConnectionException implements the IApiFault interface which has these methods. There we can get a proper message sent by Salesforce APIs.